### ISSUE TITLE
Public calendar does not display events

### ISSUE THREAD

**Issue by jurgenhaas** [nextcloud/calendar#839]
### Steps to reproduce
1. There is a shared calendar at https://cloud.tc-gottmadingen.de/index.php/apps/calendar/p/JJCjMK4zTQtHFbyG/TCG
2. Go to that link and you'll see the monthly view
3. No events are being displayed

### Expected behaviour

In that calendar there are many events, they should be displayed.

### Actual behaviour

None of the events is being displayed.

### Server configuration

Nextcloud 13.0.3 on Ubuntu 16.04 with Apache

#### Browser log

I've tried that in Chrome, Firefox and Opera, they all show the same errors in the console:

```
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src"). Source: ;undefined.
TCG:1
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src"). Source: (function (NAVIGATOR, OBJECT) {

    OBJ....
TCG:1
Content Security Policy: The page's settings blocked the loading of a resource at self ("script-src"). Source: (function (ERROR) {

    const V8_STACK_....
TCG:1
JQMIGRATE: Migrate is installed, version 1.4.0
core.js:7:542
Deprecation warning: use moment.updateLocale(localeName, config) to change an existing locale. moment.defineLocale(localeName, config) should only be used for creating a new locale See http://momentjs.com/guides/#/warnings/define-locale/ for more info.
core.js:1563:2834
Attempting to call a FullCalendar method on an element with no calendar.
vendor.js:23280:26
Source map error: request failed with status 404
Resource URL: https://cloud.tc-gottmadingen.de/core/vendor/core.js?v=6064aed3-6
Source Map URL: purify.min.js.map[Learn More]
/remote.php/dav/public-calendars/JJCjMK4zTQtHFbyG/ 
Object { body: "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<d:error xmlns:d=\"DAV:\" xmlns:s=\"http://sabredav.org/ns\">\n  <s:exception>Sabre\\DAV\\Exception\\ReportNotSupported</s:exception>\n  <s:message/>\n  <d:supported-report/>\n</d:error>\n", status: 415, xhr: XMLHttpRequest }
app.js:3346:4
```

**Comment by georgehrke** [comment]
Did this happen after an update? 

Did it never work?

**Comment by georgehrke** [comment]
@jurgenhaas Can you verify that this patch works?
https://github.com/nextcloud/server/pull/9798/files

**Comment by jurgenhaas** [comment]
I can confirm, that patch works indeed.

> Did this happen after an update?
> Did it never work?

I just tried this today and never before as I'm in the process to replace a public Google calendar with this. So I can't actually tell you the truth on those 2 questions.

But the patch is just working.

**Comment by seanbutnotheard** [comment]
I was having the same issue, and the patch works for me as well. It seems to have stopped working after the recent Nextcloud 13.0.3 update... it definitely worked correctly a couple of weeks ago.

**Comment by georgehrke** [comment]
Yes, we identified it as a regression in 13.0.3 :/ 
But we are working on providing a new release soon.

**Comment by rostchri** [comment]
The fix https://github.com/nextcloud/server/pull/9798/files is working for the empty public calendar issue, but it breaks browsing of normal calendars so this should not be the final fix ...

**Comment by georgehrke** [comment]
Was fixed with Nextcloud 13.0.4

**Comment by georgehrke** [comment]
> The fix https://github.com/nextcloud/server/pull/9798/files is working for the empty public calendar issue, but it breaks browsing of normal calendars so this should not be the final fix ...

@rostchri Browsing public and normal calendars no matter if they contain events or not works fine for me. Please open a new bug report **using the issue template** and **please provide logs**
